src/core/errorHandler.js

```diff
@@ -31,41 +36,57 @@ export class ErrorHandler {
 
 class ErrorHandlerComponent extends React.Component {
   static propTypes = {
+    autoRenderErrors: PropTypes.boolean,
```

Comment:
when it has a default value, it doesn't make sense to mark it as required because otherwise the default value would never get used.